db/models/news.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../connection.js')
+
+exports.selectTopics= ()=> {
+    return db.query(`SELECT * FROM topics`)
+    .then(({rows}) => {
+        console.log(rows)
```

Comment:
Please don't forgot to remove any console logs before commiting and pushing 🙂

Reply:
yup, sorry will do